src/Utility/ConstraintStabiliser.php

```diff
@@ -26,16 +56,16 @@ public static function stabiliseConstraints(OutputInterface $output, LibraryRele
         $parentLibrary = $releasePlanNode->getLibrary();
         $originalData = $composerData = $parentLibrary->getComposerData();
         $constraintType = $parentLibrary->getDependencyConstraint();
+        $parentName = $parentLibrary->getName();
 
         // Rewrite all dependencies.
         // Note: rewrite dependencies even if non-exclusive children, so do a global search
         // through the entire tree of the plan to get the new tag
-        $items = $releasePlanNode->getAllItems();
-        foreach ($items as $item) {
+        foreach (self::getAllItems($releasePlanNode) as $item) {
```

Comment:
Just call `$libraryRelease->getRootRelease()->getAllItems();` to get all the items

Reply:
Updated